demo/webclient/package.json

```
@@ -0,0 +1,79 @@
+{
+  "name": "mediasoup-demo-app",
+  "version": "3.0.0",
+  "private": true,
+  "description": "mediasoup demo app",
+  "author": "Iñaki Baz Castillo <ibc@aliax.net>",
+  "license": "All Rights Reserved",
+  "main": "lib/index.jsx",
```

In regards to this, I think we should use our own naming and authoring even if it is based on a certain version of mediasoup demo app. For instance:

```suggestion
  "name": "webrtc-webclient",
  "version": "1.0.0",
  "private": true,
  "description": "WebRTC web client based on mediasoup demo app v3",
  "author": "2060.io",
  "license": "MIT",
  "main": "lib/index.jsx",
```